processor/configuration/src/main/java/com/redhat/service/smartevents/processor/sources/slack/SlackSourceResolver.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.smartevents.processor.sources.slack;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.redhat.service.smartevents.infra.exceptions.definitions.user.GatewayProviderException;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.infra.models.gateways.Source;
+import com.redhat.service.smartevents.processor.GatewayConfiguratorService;
+import com.redhat.service.smartevents.processor.GatewayResolver;
+import com.redhat.service.smartevents.processor.actions.source.SourceAction;
+
+@ApplicationScoped
+public class SlackSourceResolver implements SlackSource, GatewayResolver<Source> {
+
+    @Inject
+    GatewayConfiguratorService gatewayConfiguratorService;
+
+    @Override
+    public Action resolve(Source source, String customerId, String bridgeId, String processorId) {
+        Action resolvedAction = new Action();
+        resolvedAction.setType(SourceAction.TYPE);
+
+        try {
+            resolvedAction.setParameters(Map.of(
+                    SourceAction.ENDPOINT_PARAM, getBridgeWebhookUrl(customerId, bridgeId),
+                    SourceAction.CLOUD_EVENT_TYPE_PARAM, CLOUD_EVENT_TYPE));
+        } catch (MalformedURLException e) {
```

Comment:
It could be good to log the error, WDYT?

Reply:
Done :+1:

Reply:
@kostola `GatewayProviderException` should be registered in `infra/src/main/resources/exceptionInfo.json` so that it is properly wrapped into an `Error` from our API. We have centralised error handling in `ExternalUserExceptionMapper` which should log the `Exception` rather than here.

Reply:
@robpblake see [this reply](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/663#issuecomment-1109726976)